lib/storage/file.js

```diff
@@ -372,7 +373,7 @@ File.prototype.copy = function(destination, callback) {
   }
 
   var query = {};
-  if (is.number(this.generation)) {
+  if (is.defined(this.generation)) {
```

Comment:
`is.defined` because "is this not a not a number that is not `NaN`" is, um, yeah.